lib/fs/fakefs.go

```diff
@@ -429,12 +468,20 @@ func (fs *fakefs) Rename(oldname, newname string) error {
 	fs.mut.Lock()
 	defer fs.mut.Unlock()
 
+	oldKey := filepath.Base(oldname)
```

Comment:
RemoveAll above needs case folding. Please add a test.

Reply:
Done. Thanks for pointing out!